tests/base_scraper/test_base_scraper.py

```diff
@@ -66,7 +66,15 @@ def test_download(self):
                           scraper2.download)
 
     def test_notimplementedexceptions(self):
-        scraper = mozdownload.Scraper(directory=self.temp_dir,
+        filename = 'download_test.txt'
+        # standard download
+        test_url = urljoin(self.wdir, 'download_test.txt')
+        scraper = mozdownload.DirectScraper(url=test_url,
+                                            destination=self.temp_dir,
+                                            version=None,
+                                            log_level='ERROR')
+        scraper.download()
+        scraper = mozdownload.Scraper(destination=self.temp_dir,
```

Comment:
I asked about those changes in my last review. Maybe you have missed to remove them? This is not related to the test method you placed the code into.